includes/admin/downloads/metabox.php

```diff
@@ -416,11 +416,15 @@ function edd_render_price_row( $key, $args = array(), $post_id, $index ) {
 	// Run our advanced settings now, so we know if we need to display the settings.
 	// Output buffer so that the headers run, so we can log them and use them later
 	ob_start();
-	do_action( 'edd_download_price_table_head', $post_id );
+	if ( has_action( 'edd_download_price_table_head' ) ) {
+		do_action_deprecated( 'edd_download_price_table_head', array( $post_id ), '2.8', 'edd_download_price_option_row' );
+	}
 	ob_end_clean();
 
 	ob_start();
-	do_action( 'edd_download_price_table_row', $post_id, $key, $args );
+	if ( -1 !== has_action( 'edd_download_price_table_row', 'edd_hijack_edd_download_price_table_head' ) ) {
```

Comment:
I'm not sure this is right, and this notice isn't triggering for me on SL master (above one is).

When the second parameter is included, the return value is the priority of that function. `edd_hijack_edd_download_price_table_head` is using priority `-1`, which means, for me, this function is always returning `-1`.

Reply:
Yes, I'm trying to figure out how to fix this now. Ideally, I would like to show the notice if an extension is using the hook, but allow it to ignore the hijacking function--so that notices only show for extensions.

Reply:
Let me know if you want help on it.

Reply:
That was painful, but I _think_ I got it.